ArduCopter/toy_mode.cpp

```diff
@@ -226,9 +226,9 @@ void ToyMode::update()
     bool power_button = false;
     bool left_change = false;
     
-    uint16_t ch5_in = hal.rcin->read(CH_5);
-    uint16_t ch6_in = hal.rcin->read(CH_6);
-    uint16_t ch7_in = hal.rcin->read(CH_7);
+    uint16_t ch5_in = RC_Channels::get_control_in(CH_5);
+    uint16_t ch6_in = RC_Channels::get_control_in(CH_6);
+    uint16_t ch7_in = RC_Channels::get_control_in(CH_7);
```

Comment:
No, that was unintentional. I that wouldn't even compile actually. I had this tendency of writing `get_control_in` instead of `get_radio_in` far more often then I should have. Good catch.